tests/acceptance/helpers/sharingHelper.js

```diff
@@ -325,6 +325,15 @@ module.exports = {
    * @param {string} sharer
    */
   acceptShare: async function(filename, user, sharer) {
+    /** In ocis, when a resource inside a folder is shared, the share details contains only the resource
+     in the path and not the containing folder
+     eg: if "simple-folder/subfolder" has been shared, the share details is shown as
+     path: "/subfolder"
+     */
```

Comment:
is this a known bug? if yes let's mention the issue here if not please open an issue

Reply:
yes it is. I mentioned the issue there now.